microsetta_admin/templates/create_kits.html

```diff
@@ -1,86 +1,90 @@
 {% extends "sitebase.html" %}
 {% block head %}
-
 <script src="/static/vendor/js/jquery.validate.min.js"></script>
 <script>
     $(document).ready(function(){
-        // Initialize form validation on the registration form.
-        // It has the name attribute "registration"
         $("form[name='kit_form']").validate({
-            // Specify validation rules
             rules: {
-                // The key name on the left side is the name attribute
-                // of an input field. Validation rules are defined
-                // on the right side
                 num_kits: "required",
                 num_samples: "required",
                 project_ids: "required",
-                // Make sure the form is submitted to the destination defined
-                // in the "action" attribute of the form when valid
                 submitHandler: function (form) {
                     form.submit();
                 }
             }
         });
+
+        $("#num_samples, #num_kits").on("change", function() {
+            var numSamples = $("#num_samples").val();
+            var numKits = $("#num_kits").val();
+            var totalSamples = numSamples * numKits;
```

Comment:
I don't think this is meeting the intent of the tool. The goal is to upload a CSV for each sample slot in all of the kits being created. So if you're creating two kits with one sample each, you upload one CSV with two rows. If you're creating two kits with three samples each, you upload three CSVs with two rows each.